benchmarks/src/sha3-rocc.c

```diff
@@ -8,11 +8,12 @@
 #include <stdint.h>
 #include "rocc.h"
 #include "sha3.h"
+#include "encoding.h"
 
 int main() {
 
   do {
-    printf("start basic test 1.\n");
+    printf("Start basic test 1.\n");
```

Comment:
If changing the output text, the `test-reference` outputs also be updated to match.  Otherwise `marshal test` will fail.

Reply:
Can I ignore a line in the `test-reference` or should I just run the test... get a cycle count and put it in there?

Reply:
From `FireMarshal/wlutil/test.py`, looks like marshal searches for the longest substring match in the output.  I believe this is how the Linux boot messages are implicitly ignored.

The reference has to match exactly, so:
1.  Add the cycle count as another line but omit it from the reference.
1. Remove the "Starting basic test 1" line from the start of the reference and program, maybe instead printing it at the end as "Test 1: Success".

Reply:
Since it is running in Spike I just put the direct cycle count. I think it is a sufficient solution for now? I can change it if you'd like.

Reply:
It does pass the tests this way if I run `./marshal test --spike ....json`

Reply:
The Linux version might have variations in the cycle count depending on the initramfs contents and what runs beforehand.  I'm also not sure how deterministic spike in terms of HTIF timing.

I can also handle this later myself since I have other fixes to the benchmarks to make.

Reply:
Doesn't matter to me. I just want to reference in the tutorial slides... "See how with the accelerator it gives X cycles and in the SW-only version it gives Y!" So as long as a cycle count of some kind is given then I am fine.

Reply:
As for `spike` I have no idea. My impression was that it was deterministic but I haven't tested that theory or really asked anyone.

Reply:
The original spike implementation handled fesvr as a separate `pthread`, so polling `fromhost` for any syscall was a source of non-determinism.  It might be different now.